tests/specs/send/send-btc.spec.ts

```diff
@@ -28,6 +28,17 @@ test.describe('send btc', () => {
       const details = await sendPage.confirmationDetails.allInnerTexts();
       test.expect(details).toBeTruthy();
     });
```

Comment:
I added this new test that makes sure the leading and trailing whitespace is removed from the address.

It's almost the same as the following test though ( `test('that asset value and recipient on preview match input'`) so I could merge them. 

That could help us reduce un-necessary test runs. So we would keep the next test but pass in an address with whitespaces